File: backend/src/main/java/com/uvtdorms/repository/dto/response/StudentLaundryAppointmentsDto.java
 package com.uvtdorms.repository.dto.response;


import java.time.LocalDateTime;

import com.uvtdorms.repository.entity.enums.StatusLaundry;

import lombok.Data;


@Data

 public class StudentLaundryAppointmentsDto {
     private StatusLaundry statusLaundry;
     private String washingMachineNumber;
     private String dryerNumber;
     private LocalDateTime intervalBeginDate;
}

File: frontend/uvtdorms/src/app/interfaces/student-laundry-appointments-dto.ts
import { StatusLaundry } from '../enums/status-laundry';

export interface StudentLaundryAppointmentsDto {
  statusLaundry: StatusLaundry;
  washingMachineNumber: String;
  dryerNumber: String;
  intervalBeginDate: Date;
}
